android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/AndroidPaySessionCallbacks.java

```diff
@@ -0,0 +1,12 @@
+package com.shopify.unity.buy;
+
+import com.shopify.unity.buy.models.MailingAddressInput;
+
+interface AndroidPaySessionCallbacks {
```

Comment:
I actually meant to add this file in the previous PR and opened this one to add this file in https://github.com/Shopify/unity-buy-sdk/pull/264. I'll make the change there and land that before this goes in.